fdbcli/TenantCommands.actor.cpp

```diff
@@ -764,7 +863,9 @@ CommandFactory tenantRegisterFactory("tenant",
                                                  "`list' prints a list of tenants in the cluster.\n"
                                                  "`get' prints the metadata for a particular tenant.\n"
                                                  "`configure' modifies the configuration for a tenant.\n"
-                                                 "`rename' changes the name of a tenant.\n"),
+                                                 "`rename' changes the name of a tenant.\n"
+                                                 "`lock` locks a tenant\n"
+                                                 "`unlock` unlocks a tenant\n"),
```

Comment:
Nit: for consistency, these should end in a period:

```suggestion
                                                 "`lock` locks a tenant.\n"
                                                 "`unlock` unlocks a tenant.\n"),
```